src/main/java/com/twilio/rest/video/v1/Composition.java

```diff
@@ -202,9 +202,9 @@ private Composition(@JsonProperty("account_sid")
                         @JsonProperty("date_created")
                         final String dateCreated,
```

Comment:
Why is this a String if the others are DateTime objects?

Reply:
See other comment.

Reply:
👍